.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 language: go
 
 go:
-- "1.11"
+- "1.15"
```

Comment:
Go version should not be more than `1.13`

Reply:
Changed to 1.13

Reply:
This needs to be 1.15 now, a few things changed in the past couple of months.

Reply:
There is no more `.travis.yml` file in this repo, so skipping this one.